File: charactermanager/backend/src/main/java/ai/ragu/charactermanager/pojo/AbilityScore.java
package ai.ragu.charactermanager.pojo;

import lombok.Getter;
import lombok.Setter;

@Entity
@Table(name = "AbilityScore")
@Getter
@Setter
public class AbilityScore {

    @Id
    @Column(name = "id")
    private String id;

    @Column(name = "name")
    private String name;

    @Column(name = "description")
    private String description;

}

File: charactermanager/backend/src/main/java/ai/ragu/charactermanager/pojo/AbilityScore.kt
package ai.ragu.charactermanager.pojo

import AbstractJpaPersistable
import javax.persistence.Column
import javax.persistence.Entity
import javax.persistence.Id
import javax.persistence.Table

@Entity
@Table(name = "ability_score")
class AbilityScore : AbstractJpaPersistable<Long>() {
    @Id
    @Column(nullable = false)
    lateinit var id: String

    @Column(nullable = false)
    lateinit var name: String

    @Column(nullable = true)
    lateinit var description: String
}